pkg/security/grantvmgroupaccess_test.go

```diff
@@ -0,0 +1,106 @@
+package security
+
+import (
+	"io/ioutil"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"regexp"
+	"strings"
+	"testing"
+)
+
+// TestGrantVmGroupAccess verifies for the three case of a file, a directory,
+// and a file in a directory that the appropriate ACEs are set, including
+// inheritance in the second two examples. These are the expected ACES. Is
+// verified by running icacls and comparing output.
+//
+// File:
+// S-1-15-3-1024-2268835264-3721307629-241982045-173645152-1490879176-104643441-2915960892-1612460704:(R,W)
+// S-1-5-83-1-3166535780-1122986932-343720105-43916321:(R,W)
+//
+// Directory:
+// S-1-15-3-1024-2268835264-3721307629-241982045-173645152-1490879176-104643441-2915960892-1612460704:(OI)(CI)(R,W)
+// S-1-5-83-1-3166535780-1122986932-343720105-43916321:(OI)(CI)(R,W)
+//
+// File in directory (inherited):
+// S-1-15-3-1024-2268835264-3721307629-241982045-173645152-1490879176-104643441-2915960892-1612460704:(I)(R,W)
+// S-1-5-83-1-3166535780-1122986932-343720105-43916321:(I)(R,W)
+
+func TestGrantVmGroupAccess(t *testing.T) {
+	f, err := ioutil.TempFile("", "gvmgafile")
+	if err != nil {
+		t.Fatal(err)
+	}
+	defer f.Close()
+	defer os.Remove(f.Name())
```

Comment:
Oh Windows :).

You need to switch the order here. The `os.Remove` will fail because the `f.Close()` hasn't run yet. Maybe make it explicit by a single function
```go
defer func() {
    f.Close()
    os.Remove(f.Name())
}()
```

Reply:
Fixed. Push coming soon...